source-gallery/src/main/java/com/google/android/apps/muzei/gallery/GallerySettingsActivity.java

```diff
@@ -95,7 +93,7 @@
 import static com.google.android.apps.muzei.gallery.GalleryArtSource.EXTRA_FORCE_URI;
 
 public class GallerySettingsActivity extends AppCompatActivity
-        implements LoaderManager.LoaderCallbacks<Cursor> {
+        implements LoaderManager.LoaderCallbacks<Cursor>, GalleryImportPhotosDialogFragment.OnRequestContentListener {
```

Comment:
Put this on a newline. Perhaps move `implements` to the first line to make the formatting a bit better.